.landscaper/gardenlet/example/imports.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: imports.gardenlet.landscaper.gardener.cloud/v1alpha1
+kind: LandscaperGardenletImport
+runtimeCluster:
+  apiVersion: landscaper.gardener.cloud/v1alpha1
+  kind: Target
+  spec:
+    type: landscaper.gardener.cloud/kubernetes-cluster
+    config:
+      kubeconfig: |
+        apiVersion:...
+        # here goes the kubeconfig of the target cluster
+
+gardenCluster:
+  spec:
+    type: landscaper.gardener.cloud/kubernetes-cluster
+    config:
+      kubeconfig: |
+        ---
+        apiVersion: v1
+        kind: Config
+      # ... <please insert your kubeconfig here
+
+# optional image vector. Please check docs/deployment/image_vector.md.
+#imageVectorOverwrite: ""
+
+# optional image vector for the components deployed by the Gardenlet itself. Please check docs/deployment/image_vector.md.
+#componentImageVectorOverwrites: ""
+
+# optional backup provider for the Seed cluster registered by the Gardenlet
+# required when gardenlet.componentConfiguration.seedConfig.backup.secretRef is not set
+# secret is deployed into the virtual Garden cluster
+seedBackup:
+  provider: aws
+  credentials:
```

Comment:
I don't see those fields in the blueprint's JSON schema.

Reply:
The naming in the blueprint was incorrect. Update it.